Sources/Keychain/KeychainHelper.swift

```diff
@@ -53,11 +62,13 @@ enum KeychainHelper {
 
     let status = SecItemAdd(attributes, nil)
```

Comment:
@Itaybre shouldn't this be changed as well? Or should this function not have a completion block?